universal-application-tool-0.0.1/app/views/admin/questions/QuestionConfig.java

```diff
@@ -84,16 +85,19 @@ public static ContainerTag buildQuestionConfig(QuestionType type, QuestionForm q
             .addMultiOptionQuestionFields(form)
             .addMultiSelectQuestionValidation(form)
             .getContainer();
-      case DROPDOWN:
+      case NUMBER:
+        return config
+            .setId("number-question-config")
+            .addNumberQuestionConfig((NumberQuestionForm) questionForm)
+            .getContainer();
       case RADIO_BUTTON:
         return config
             .setId("single-select-question-config")
             .addMultiOptionQuestionFields((MultiOptionQuestionForm) questionForm)
             .getContainer();
-      case NUMBER:
-        return config.setId("number-question-config").addNumberQuestionConfig().getContainer();
-      case REPEATER: // fallthrough intended
+      case DROPDOWN: // fallthrough intended - no options
```

Comment:
suggestion (blocking): DROPDOWN should fall through to RADIO_BUTTON (they are both single-select questions). Would you mind adding a comment mentioning this fall through too?

Reply:
Oops - thank you for catching this! I misunderstood the way switch statement fall-throughs work. Yes, a comment will help.